pkg/plugin/go_plugin_installer.go

```diff
@@ -48,7 +48,7 @@ func (g *GoPluginInstaller) CheckVersion(ver *version.Version) error {
 }
 
 func (g *GoPluginInstaller) Install() error {
-	packageName := fmt.Sprintf("github.com/confluentinc/cli/v3-plugins/%s@latest", g.Name)
+	packageName := fmt.Sprintf("github.com/confluentinc/cli-plugins/%s@latest", g.Id)
```

Comment:
This looks like a bug fix. Should we make a separate PR for this so users don't have to wait for v4 to get these changes?

Reply:
fixed it on main. https://github.com/confluentinc/cli/pull/2763